docs/_steps/saving_workflow_step.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Saving the step configuration
+lang: en
+slug: saving-steps
+order: 4
+---
+
+<div class='section-content'>
+
+When the workflow step's configuration has been saved (using the step configuration modal from the `edit` callback), your app will listen for the `view_submission` event. The method assigned to the `save` property of the `WorkflowStep` configuration object passed in during instantiation will run when this event occurs.
+
+Once the configuration for the workflow step has been determined, builders often use that configuration to craft the custom outputs and behavior that occurs when the end user executes the step.
```

Comment:
I don't really understand how this relates to the content around it. Could it just be removed?